pygeo/pyBlock.py

```diff
@@ -858,7 +855,31 @@ def projectPoints(self, x0, checkErrors, embTol, eps, nIter):
         v0 = 0.0
         w0 = 0.0
 
+        # If we are only interested in interior points, we skip projecting exterior points to save time.
+        # We identify exterior points by checking if they are outside the convex hull of the control points.
+        # A point can be inside the convex hull but still outside the FFD volume(s).
+        # In this case, the point is identified as an exterior point by the projection, which is more costly.
```

Comment:
Could this be rephrased to something along the lines of "Only in this case we rely on the more costly projection approach to identify the exterior points."? I got confused for a sec with the current phrasing, but probably it's just a me problem

Reply:
Rephrased